prisma/schema.prisma

```diff
@@ -323,3 +323,9 @@ model VerificationToken {
 
     @@unique([identifier, token])
 }
+
+model NewUser {
+    id    String @id
+    name  String
+    email String @unique
```

Comment:
Do we need another field to store the shibboleth groups, `unscopedAffiliations`, i.e. staff, student etc. ?

Reply:
How we store them depends I suppose on whether a user can belong to multiple groups/`unscopedAffiliations` with the same account, i.e. would they use the same logins as a student and staff. 

If a user can belong to multiple and we need to store them we can create a `UserGroups` or `UserAffiliations` table to store this information.

We probably don't need to store them on the user model, as long as we create the appropriate `StudentDetails`, `SupervisorDetails` and `AdminInSpace` rows when a user gets invited to a space.

Reply:
Ahhhh okay, that makes more sense to do it that way.